src/handlers/index.ts

```diff
@@ -19,7 +22,12 @@ const functions = {
   getConfig: getConfigHandler,
   updateConfig: updateConfigHandler,
   getYnabAccountData,
+  electronGoogleLogin: googleLoginHandler,
+  validateToken: validateTokenHandler,
```

Comment:
Token is a generic term and can be relevant to other things. validateToken  -> validateGoogleToken